x-pack/solutions/observability/plugins/streams/server/routes/streams/crud/read_stream.ts

```diff
@@ -30,14 +32,25 @@ export async function readStream({
   assetClient: AssetClient;
   streamsClient: StreamsClient;
   scopedClusterClient: IScopedClusterClient;
-}): Promise<IngestStreamGetResponse> {
-  const [streamDefinition, dashboards, ancestors, dataStream] = await Promise.all([
+}): Promise<StreamGetResponse> {
+  const [streamDefinition, dashboards] = await Promise.all([
     streamsClient.getStream(name),
     assetClient.getAssetIds({
       entityId: name,
       entityType: 'stream',
       assetType: 'dashboard',
     }),
+  ]);
+
+  if (isGroupStreamDefinition(streamDefinition)) {
+    return {
+      stream: omit(streamDefinition, 'name'),
```

Comment:
Why do we omit the name here?

Reply:
That might have been something left over from a merge when the name was omitted. I just looked at the underlying classes and it SHOULD NOT be omitted. Fixing that now.